cfgov/cfgov/urls.py

```diff
@@ -455,23 +446,41 @@ def empty_200_response(request, *args, **kwargs):
         view_answer,
         name='ask-spanish-answer'
     ),
-    re_path(
+
+    # If 'ELASTIC_SEARCH_DSL' is True, use elasticsearch7.
+    flagged_re_path(
+        'ELASTIC_SEARCH_DSL',
+        r'^(?P<language>es)/obtener-respuestas/buscar/$',
+        ask_search_es7,
+        name='ask-search-es'
```

Comment:
You'll need to provide the original view as a [`fallback` for these flagged urls](https://cfpb.github.io/django-flags/api/urls/#flagged_re_pathflag_name-route-view-kwargsnone-namenone-statetrue-fallbacknone). Something like:

```python
    flagged_re_path(
        'ELASTIC_SEARCH_DSL',
        r'^(?P<language>es)/obtener-respuestas/buscar/$',
        ask_search_es7,
        fallback=ask_search,
        name='ask-search-es'
    ),
```

Reply:
Done

Reply:
Added fallback to flagged_re_path.